### PR TITLE
Improve spacing of PR header buttons

### LINKED ISSUE #4277: `view-last-pr-deployment` is not spaced correctly on editable PRs
Screen Shot 4

In my case the PR was https://github.com/npmgraph/npmgraph/pull/75

It's only missing a single space character

### PR DESCRIPTION
**Pull request by kidonng**
Fix #4277

## Test URLs

See #4089

## Screenshot

image

image

### CODE REVIEW

**Comment by fregante** [comment]
We just need to add a single space character, unconditionally. Mr-1 is not exactly the same I think

**Comment by kidonng** [comment]
Does it work? I tried and not luck 😅

File: source/features/view-last-pr-deployment.tsx [hunk 1]
```diff
@@ -14,9 +14,10 @@ function init(): void {
 	}
 
 	const {href} = select.last<HTMLAnchorElement>(deploymentSelector)!;
-	select('.gh-header-actions')!.prepend(
+	const headerActions = select('.gh-header-actions')!;
+	headerActions.prepend(
 		<a
-			className="rgh-last-deployment btn btn-sm d-none d-md-block"
+			className={`rgh-last-deployment btn btn-sm d-none d-md-block ${select.exists('[aria-label="Edit Pull Request title"]', headerActions) ? 'mr-1' : ''}`}
```
Comment (fregante) [inline source/features/view-last-pr-deployment.tsx:20]: I don't mind having a little extra space to save all of this code here.

This diff should just include an unconditional `mr-1`

Comment (kidonng) [reply source/features/view-last-pr-deployment.tsx:20]: What do you think about https://github.com/sindresorhus/refined-github/issues/4277#issuecomment-826358707?

Perhaps add a 4px left margin to `Open With` in `refined-github.css` so it looks better and consistent.

---

Also I find that on the issue page the spacing is 8px:

image

vs PR page:

image

**Comment by kidonng** [comment]
I added new screenshots in PR description.

**Comment by fregante** [comment]
If you’re going that route, we only need a CSS rule, no extra classes. I think left:8px and right:4px

**Comment by kidonng** [comment]
That won't work on non-editable PR, the spacing will be 4px from "Open With".

File: source/refined-github.css [hunk 2]
```diff
@@ -149,3 +149,9 @@ pr-branches
 	box-shadow: var(--color-toast-shadow);
 	z-index: 2147483647;
 }
+
+/* Correct spacing between "Edit" and "Open with" button on PR page */
```
Comment (fregante) [inline source/refined-github.css:153]:
```suggestion
/* Space out "Edit" and "Open with" buttons on PR page, also for `view-last-pr-deployment` #4281 */
```

**Comment by fregante** [comment]
# 💯

**Pull request merged**